LayoutTests/imported/w3c/web-platform-tests/css/css-inline/text-box-trim/text-box-edge-computed-expected.txt

```diff
@@ -1,5 +1,5 @@
 
-FAIL Property text-box-edge value 'leading' assert_equals: expected "leading" but got "leading leading"
+PASS Property text-box-edge value 'leading'
 FAIL Property text-box-edge value 'text' assert_equals: expected "text" but got "text text"
 FAIL Property text-box-edge value 'ex' assert_equals: expected "ex" but got "ex text"
 FAIL Property text-box-edge value 'cap' assert_equals: expected "cap" but got "cap text"
```

Comment:
You could fix the other ones pretty easily :) 

```
if (textBoxEdge.under == TextBoxEdgeType::Text)
    return createConvertingToCSSValueID(textBoxEdge.over);
```

Reply:
thanks!